Assets/MixedRealityToolkit.Tools/MSBuild/Scripts/AssemblyDefinitionInfo.cs

```diff
@@ -93,6 +180,60 @@ public void Validate(IEnumerable<CompilationPlatformInfo> availablePlatforms)
 
             TestAssembly = optionalUnityReferences?.Contains(TestAssembliesReference) ?? false;
         }
+
+        /// <summary>
+        /// Gets the source files for this assembly definition.
+        /// </summary>
+        /// <returns>The array of <see cref="SourceFileInfo"/>.</returns>
+        public SourceFileInfo[] GetSources()
+        {
+            if (sourceFiles == null)
+            {
+                List<SourceFileInfo> sourceFiles = new List<SourceFileInfo>();
+                if (assembly != null)
+                {
+                    foreach (string file in assembly.sourceFiles)
```

Comment:
>file [](start = 36, length = 4)

Super nit: sourceFile - "file" as a var already exists in the class context, and when paired with the reference down below (i.e. file.Directory in the else statement) it seeeeems a little confusing